src/pricing/uniswap.ts

```diff
@@ -1,12 +1,13 @@
 // pricing for uniswap traded tokens and uniswap LP tokens
 
-import { Address, BigInt, BigDecimal, log } from '@graphprotocol/graph-ts'
+import { Address, BigInt, BigDecimal, log, ethereum, dataSource } from '@graphprotocol/graph-ts'
```

Comment:
It doesn't look like either of these are being used

Reply:
fixed :+1: